client/src/test/java/com/paypal/selion/TestServerListener.java

```diff
@@ -25,7 +25,7 @@ public void onStart(ISuite suite) {
         try {
             TestServerUtils.startServer();
         } catch (Exception e) { // NOSONAR
-            e.printStackTrace();
+            System.err.println(" Unable to start test server");
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [System.err.println is used](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1537384634/issues/source?bid=2928425&fileBranchId=2989551#l28)